src/Controller/ResetPasswordControllerTrait.php

```diff
@@ -90,4 +90,13 @@ private function getSessionService(): SessionInterface
 
         return $request->getSession();
     }
+
+    /**
+     * Generate a fake token to be used in the session if needed to prevent
+     * revealing if a user exists.
+     */
+    private function getFakeToken(int $tokenLifetime): ResetPasswordToken
+    {
+        return new ResetPasswordToken('fake-token', new \DateTimeImmutable(\sprintf('+%d seconds', $tokenLifetime)), \time());
+    }
```

Comment:
So, with my suggestion, this would not be needed.

Reply:
agreed